### PR TITLE
Include ENV variables for configure queue manager

### PR DESCRIPTION
**Pull request by caleeli**
## Solution
Add a ENV variable to configure the number of processes in Horizon config

## How to Test
Install or deploy  PM4, and use the env variables to setup the horizon parameters.
Start o Restart the horizon manager to load/reload the settings.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-6283

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

**Review by epdl — APPROVED** [review]
Looks good to me

File: config/horizon.php [hunk 1]
```diff
@@ -145,59 +145,83 @@
             'supervisor-bpmn' => [
                 'connection' => 'redis',
                 'queue' => ['bpmn'],
-                'balance' => 'simple',
-                'processes' => 16,
-                'tries' => 1,
-                'timeout' => 3600,
+                'balance' => env('PM4_HORIZON_SUPERVISOR_BPMN_BALANCE', 'simple'),
+                'processes' => env('PM4_HORIZON_SUPERVISOR_BPMN_PROCESSES', 4),
```
Comment (epdl) [inline config/horizon.php:149]: Reviewing the documentation, looks like 'processes' are not compatible with 'minProcesses' and 'maxProcesses', we need to pick one only.

**Review by epdl — CHANGES_REQUESTED** [review]

File: config/horizon.php (hunk 1 above)
Comment (nolanpro) [reply config/horizon.php:149]: @epdl I think this is based on the `balance` setting. If balance is simple or false, the `processes` setting is used. If balance is `auto`, then `processes` is ignored and the `minProcesses` and `maxProcesses` is used.

**Review by nolanpro — CHANGES_REQUESTED** [review]
@caleeli 
- `balanceMaxShift` and `balanceCooldown` are not supported until laravel 8 so we can remove those
- No matter what the balance setting is, When the `max_processes` key exists in the config, it will overwrite `processes` (see https://github.com/laravel/horizon/blob/3.0/src/ProvisioningPlan.php#L155) So we can get rid of `processes` and update the max_processes description in .env.md

**Review by nolanpro — APPROVED** [review]

**Review by ryancooley — APPROVED** [review]

**Pull request merged**